src/ui/Templates/Template.ts

```diff
@@ -212,6 +212,7 @@ export class Template implements ITemplateProperties {
     // In some circumstances (eg: locker service in SF), with strict Content-Security-Policy, eval / new Function are not allowed by the browser.
     // Try to use the eval method, if possible. Otherwise fallback to a mechanism where we will try to parse/evaluate the condition as a simple string.
     try {
+      // Eval
       this.condition = new Function('obj', 'with(obj||{}){return ' + condition + '}');
     } catch (e) {
       this.conditionToParse = condition;
```

Comment:
We have two instances where `new Function` is evaluating a string. In this case here, we have a fallback mechanism.

I did not explore the limitations of the fallback, but if they are minimal, we could make the fallback the default mechanism, and remove the `eval` expressions.